### PR TITLE
 changelog Version 0.29.0: July 15, 2020

### PR DESCRIPTION
**Pull request by elahehrashedi**
# C/C++ for Visual Studio Code Change Log

## Version 0.29.0: July 15, 2020

### CODE REVIEW

File: Extension/CHANGELOG.md [hunk 1]
```diff
@@ -53,6 +47,7 @@
 * Fix find <miDebuggerPath>.exe in Windows path [#3076](https://github.com/microsoft/vscode-cpptools/issues/3076)
   * [PR MIEngine#1001](https://github.com/microsoft/MIEngine/pull/1001)
 
+
```
Comment (sean-mcmanus) [inline Extension/CHANGELOG.md]: Can you remove this extra newline?

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]

**Review by michelleangela — APPROVED** [review]

**Review by sean-mcmanus — APPROVED** [review]

File: Extension/CHANGELOG.md [hunk 2]
```diff
@@ -39,12 +27,18 @@
...
   * Matt Schulte (@schultetwin1)
-* Fix IntelliSense process crashes. [#5584](https://github.com/microsoft/vscode-cpptools/issues/5584), [#5629](https://github.com/microsoft/vscode-cpptools/issues/5629)
 * Fix Find All References and Rename when multiple references are on the same line. [#5568](https://github.com/microsoft/vscode-cpptools/issues/5568)
+* Fix IntelliSense process crashes. [#5584](https://github.com/microsoft/vscode-cpptools/issues/5584), [#5629](https://github.com/microsoft/vscode-cpptools/issues/5629)
 * Fix an add/remove workspace folder crash. [#5591](https://github.com/microsoft/vscode-cpptools/issues/5591)
 * Fix default build tasks failing on Windows if the compiler isn't on the PATH. [#5604](https://github.com/microsoft/vscode-cpptools/issues/5604)
 * Fix updating `files.associations` and .C files being associated with C instead of C++. [#5618](https://github.com/microsoft/vscode-cpptools/issues/5618)
 * Fix IntelliSense malfunction when RxCpp is used. [#5619](https://github.com/microsoft/vscode-cpptools/issues/5619)
+* Ignore "screen size is bogus" error when debugging. [PR #5669](https://github.com/microsoft/vscode-cpptools/pull/5669)
```
Comment (bobbrow) [inline Extension/CHANGELOG.md:36]: Did the "bug fixes" label get removed?

Comment (bobbrow) [reply Extension/CHANGELOG.md:36]: nevermind.  It was collapsed in the web view.  I see it now.

File: Extension/CHANGELOG.md [hunk 3]
```diff
@@ -1,31 +1,19 @@
 # C/C++ for Visual Studio Code Change Log
 
-## Version 0.29.0-insiders2: July 8, 2020
+## Version 0.29.0: July 15, 2020
 ### New Features
-* Added support for LogMessage Breakpoints for debug type `cppdbg`. [#1013](https://github.com/microsoft/MIEngine/pull/1013)
-* Switch to using the VS Code Semantic Tokens API for semantic colorization. [PR #5401](https://github.com/microsoft/vscode-cpptools/pull/5401)
+* Add Doxygen comment support (to tooltip display of hover, completion, and signature help). [#658](https://github.com/microsoft/vscode-cpptools/issues/658)
+  * The way comments are formatted is controlled by the C_Cpp.simplifyStructuredComments setting."
```
Comment (sean-mcmanus) [inline Extension/CHANGELOG.md:6]: There's an extra " -- we can fix that later.

**Pull request merged**